website/www/site/content/en/about/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "About"
+---
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# About
+
+## Why Apache Beam?
+
+Apache Beam is an open-source, unified programming model for batch and streaming data processing pipelines that simplifies large-scale data processing dynamics. Thousands of organizations around the world choose Apache Beam due to its unique data processing features, proven scale, and powerful yet extensible capabilities.
+
+<div style="display: flex; justify-content: center">
+  <img src="/images/batch_stream_processing.png" alt="Batch and Stream Processing">
+</div><br /><br />
+
+Apache Beam is the future of data processing because it provides:<br /><br />
+
+- <u>Powerful Abstraction</u>
+
+The Apache Beam model offers powerful abstractions that insulate you from low-level details of distributed data processing, such as coordinating individual workers, reading from sources and writing to sinks, etc. The pipeline abstraction encapsulates all the data and steps in your data processing task. You can think of your data processing tasks in terms of these abstractions. The higher level abstractions neatly separate data from runtime characteristics and simplify the mechanics of large-scale distributed data processing. You focus on creating value for customers and business while the Dataflow model handles the rest.<br /><br />
+
+- <u>Unified Batch and Streaming Programming Model</u>
+
+Apache Beam provides flexibility to express the business logic just once and execute it for both batch and streaming data pipelines, on-premises via OSS runners or in the cloud via managed services such as Google Cloud Dataflow or AWS Kinesis Data Analytics.  Apache Beam unifies multiple data processing engines and SDKs around its distinctive Beam model. This offers a way to easily create a large-scale common data infrastructure across different applications that consume the data.<br /><br />
```

Comment:
seems like there's an extra space here: "... Analytics.  Apache Beam unifies ..." 

You can fix it by formatting the code.

Reply:
done